frontend/src/components/object/ObjectTable.vue

```diff
@@ -85,18 +96,47 @@ async function showPermissions(objectId: string) {
 
 watch(getObjects, async () => {
   clearSelected();
+  loadLazyData(lazyParams);
+});
 
-  // Filter object cache to this specific bucket
-  const objs: Array<COMSObjectDataSource> = getObjects.value.filter(
-    (x: COMSObject) => x.bucketId === props.bucketId
-  ) as COMSObjectDataSource[];
+onMounted(() => {
+  loading.value = true;
 
-  tableData.value = objs.map((x: COMSObjectDataSource) => {
-    x.lastUpdatedDate = x.updatedAt ?? x.createdAt;
-    return x;
-  });
+  lazyParams.value = {
+    first: 0,
+    rows: 10,
+    sortField: null,
+    sortOrder: null,
+    filters: filters.value
+  };
+  loadLazyData(lazyParams);
 });
 
+const loadLazyData = (event: any) => {
+  lazyParams.value = { ...lazyParams.value, first: event?.first || first.value };
+  objectService
+    .searchObjects({
+      bucketId: props.bucketId ? [props.bucketId] : undefined,
+      deleteMarker: false,
+      latest: true,
+      permissions: true,
+      page: lazyParams.value.page ? lazyParams.value.page + 1 : 1,
+      limit: lazyParams.value.rows
+    })
+    .then((r: any) => {
+      loading.value = true;
```

Comment:
maybe this should be moved to start of function (line 116). and maybe removed from the onMounted() function

Reply:
It has been readjusted.